adoc/deployment-azure.adoc

```diff
@@ -0,0 +1,204 @@
+== Deployment on Microsoft Azure
+
+Deployment on Microsoft Azure is currently a tech preview.
+
+.Preparation Required
+[NOTE]
+====
+You must have completed <<deployment.preparations>> to proceed.
+====
+
+You will use {tf} to deploy the whole infrastructure described in
+<<architecture-azure>>. Then you will use the `skuba` tool to bootstrap the
+{kube} cluster on top of it.
+
+
+[[architecture-azure]]
+=== AZURE Deployment
+
+The AZURE deployment created by our {tf} template files leads to the
+creation of the infrastructure described in the next paragraphs.
+
+==== Network
+
+All of the infrastructure is created inside of a user specified AZURE region.
+The resources are currently all located inside of the same availability
+zone. All the nodes are placed inside of the same virtual network, within the same
+subnet.
```

Comment:
This is wrong, the TF file now supports multiple availability zones if properly configured by the user

Reply:
updated